packages/tasit-action/src/erc721/ERC721Full.test.js

```diff
@@ -163,12 +165,15 @@ describe("TasitAction.ERC721.ERC721Full", () => {
 
     it("should transfer an approved token", async () => {
       erc721 = new ERC721Full(ERC721_FULL_ADDRESS, ana);
+      erc721.setWallet(ana);
```

Comment:
Why wasn't this needed before?

Reply:
Oh, good catch.
I've added this line for debugging and forgot to remove.
Done.

Reply:
Ah ok, glad we caught this. Whenever I add debugging lines I like to add `// TODO: Remove me before merging`

Reply:
Resolving this